addons/context_actions/functions/fnc_healPlayers.sqf

```diff
@@ -0,0 +1,27 @@
+/*
+ * Author: 3Mydlo3
+ * Heals players in given selection. Works for players in vehicles.
+ *
+ * Arguments:
+ * N: Selected Objects <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player1, player2, player3] call zen_context_actions_fnc_healPlayers
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private _units = [];
+{
+    _units append crew _x;
```

Comment:
Shouldn't this be using `fullCrew`? As in if you use it on a vehicle, currently, you would only heal the - driver, gunner, commander. But what about the passengers in the vehicle?

And for that matter, if you select a single unit (from the unit tree) and open the context menu on him and press heal, then it will heal everyone in the vehicle, even though you wanted only 1 unit to get healed.

Reply:
It won't heal passengers, that's true, I'll fix this.

But I don't agree on the second part. If I select 1 guy in a vehicle (using unit tree), `curatorSelected` returns only this selected unit, and later on `crew _x` also returns this one unit. Maybe I cannot see something obvious, feel free to correct me.

Reply:
🤔 I’ll take a look later in game. Maybe I’m confusing something.

Reply:
I performed further tests and now I don't agree with first part either. `crew` according to wiki returns "crew in order [driver,gunner,commander,turrets,cargo]" and my tests confirm that, so everyone on board would be healed (as long as it's player).

There might be just one issue, when we select vehicle and it's crew and then click heal, as everyone will be healed twice, but it can be easily fixed by adding `arrayIntersect`.

Reply:
Yeah, you're right, it looks like I was jumping to conclusions from the first line of the BIKI page :smile: 
And yeah, that should be addressed.